builtins/amp-img.md

```diff
@@ -164,7 +164,7 @@ If you want your image to scale as the window is resized but up to a maximum wid
 
 #### The difference between responsive and intrinsic layout
 
-Both the `responsive` and `intrinsic` layouts create an image that will scale automatically.  The main difference is that the `intrinsic` layout uses an SVG image as its scaling element.  This will make it behave in the same way as a standard HTML image while retaining the benefit of the browser knowing the image size on initial layout. The `intrinsic` layout will have an intrinsic size and will inflate a floated `div` until it reaches either the natural image size or a CSS constraint like `max-width`. The `responsive` layout will render 0x0 in a floated `div` because it takes its size from the parent, which has no natural size when floated.
+Both the `responsive` and `intrinsic` layouts create an image that will scale automatically.  The main difference is that the `intrinsic` layout uses an SVG image as its scaling element.  This will make it behave in the same way as a standard HTML image while retaining the benefit of the browser knowing the image size on initial layout. The `intrinsic` layout will have an intrinsic size and will inflate a floated `div` until it reaches either the natural image size (as defined by the width and height attributes passed to the amp-img, not by the natural size of the image) or a CSS constraint like `max-width`. The `responsive` layout will render 0x0 in a floated `div` because it takes its size from the parent, which has no natural size when floated.
```

Comment:
I'd clarify this to:
...until it reaches either the image size defined by the `width` and `height` attributes passed to the `amp-img` (not by the natural size of the image) or a CSS constraint such as `max-width`.

Reply:
I'm glad. I was hoping we could make this explicit. =) I will make this change shortly.